Makefile

```diff
@@ -305,6 +305,17 @@ prow-monitoring:
 	make -C cluster/ci/monitoring prow-monitoring-deploy
 .PHONY: prow-monitoring
 
+build-farm-consistency:
```

Comment:
I don't think we need this complexity for a `cp`

Reply:
Done.

Reply:
Removed.